docs/flatcar-linux/aws.md

```diff
@@ -208,7 +208,7 @@ Check the [variables.tf](https://github.com/kinvolk/lokomotive-kubernetes/blob/m
 | dns_zone | AWS Route53 DNS zone | "aws.example.com" |
 | dns_zone_id | AWS Route53 DNS zone id | "Z3PAABBCFAKEC0" |
 | ssh_authorized_key | SSH public key for user 'core' | "ssh-rsa AAAAB3NZ..." |
-| asset_dir | Path to a directory where generated assets should be placed (contains secrets, used to destroy the cluster), cannot be a relative path | "/home/user/.secrets/clusters/tempest" |
+| asset_dir | Path to a directory where generated assets should be placed (contains secrets, used to destroy the cluster). | "./assets" |
```

Comment:
I will check with git-blame who wrote it can't be a relative path, and ask just in case we are missing something

Reply:
@surajssd feel free to close this if you think testing is enough instead of pinging :)

Reply:
Maybe write, can be relative or absolute but if `~` is included, path expanding via `pathexpand("yourpath")` is needed.